### DISCUSSION TITLE
24 neopixel ring

### DISCUSSION THREAD

**Discussion by seattlebuster** [gpstar81/GPStar-proton-pack#38] [Ideas]
I'm going to attempt the 24 neopixel ring,  is it as simple as changing the code to reflect that number in the ring?

**Comment by gpstar81** [comment]
Good question, in theory it should if you change:

#define CYCLOTRON_NUM_LEDS 35

that to 24. However it may spin too fast, so you may need to use lower delay multipliers on these:

const int i_2021_inner_delay = 10;
const int i_1984_inner_delay = 6;

As those are configured for 35 rings. The lower the number the slower it will go. So maybe try a 8 for 2021 and 4 for 1984. Adjust as needed. Let me know how it looks and what numbers you find looked the best for a 24 ring, then it can be added to the code.

**Reply by Yanks1023** [reply]
Looks like the 35 ring is temporarily out of stock. Any suggestions for substitutions?

**Reply by gpstar81** [reply]
I suggest keep looking for a 35. You could try a 24 but the diameter of the 35 is perfect.

**Reply by Yanks1023** [reply]
Found this one...should still work with the code? 

https://www.amazon.com/WS2812B-Addressable-Modules-WS2812-Circle/dp/B09NGVJJWJ/ref=sr_1_10?crid=2RIE5XZ1ZUZ9H&keywords=35+Bits+WS2812+Addressable+LED+Ring&qid=1679944779&s=hi&sprefix=35+bits+ws2812+addressable+led+ring%2Ctools%2C162&sr=1-10

**Reply by gpstar81** [reply]
Yup

**Reply by Yanks1023** [reply]
I'm going to grab one of these rings. Is it wired the same way as NEO-CYC?

**Reply by gpstar81** [reply]
Yes it is wired the same.

**Comment by DustinGrau** [comment]
Closing this out as the new EEPROM menu provides a standard option for 24-pixel inner cyclotron rings as well as other popular sizes.
